src/components/offer-card/offer-actions/OfferActions.tsx

```diff
@@ -9,8 +10,32 @@ import TurnedInNot from '@mui/icons-material/TurnedInNot'
 import AppButton from '~/components/app-button/AppButton'
 
 import { styles } from '~/components/offer-card/offer-actions/OfferActions.styles'
+import { VariantEnum } from '~/types'
+
+interface OfferActionsProps {
+  price: string
+  isBookmarked: boolean
+  onBookmarkClick: () => void
+  buttonActions: { label: string; handleClick: () => void }[]
```

Comment:
maybe create interface for buttonAction and move it to types folder for offer to make it cleaner

Reply:
Fixed